deker/integrity.py

```diff
@@ -170,38 +170,43 @@ def check_arrays_locks(self, collection: Collection) -> None:
         if self.stop_on_error and self.errors:
             raise DekerIntegrityError(self._parse_errors())
 
-    def check(self, collection: Collection) -> None:
-        """Check if Arrays or VArray in Collection are valid.
+    def _check_v_arrays(self, collection: Collection, v_arrays: Union[ArrayManager, Optional[VArrayManager]]) -> None:
+        """Check if Arrays or VArray in Collection are initializing.
 
         :param collection: Collection to be checked
+        :param v_arrays: DataManager to get arrays or varrays from collection
```

Comment:
I think it's better to rename it clearly to `data_manager` if it may iterate over any collection.

Reply:
+1